lib/Doctrine/ORM/EntityNotFoundException.php

```diff
@@ -30,8 +30,13 @@ class EntityNotFoundException extends ORMException
     /**
      * Constructor.
      */
-    public function __construct($class)
+    public function __construct($class, array $id)
     {
-        parent::__construct("Entity of type '{$class}' was not found.");
+        $ids = array();
+        foreach ($id as $key => $value) {
+            $ids[] = "{$key}({$value})";
```

Comment:
Feedback on the formatting welcome. Currently the message will look like this: `Entity of type 'MyClass' for ID id1(42), id2(23)`